### PR TITLE
⚡️  (INT-1191): PublicAPI Login Support for OAuth and Password

### PR DESCRIPTION
**Pull request by fahimmahmoodmir**

### CODE REVIEW

**Comment by fahimmahmoodmir** [comment]
Screen-Shot-PublicAPI

Output Image ^^

File: pages/documents/ConversationalAI/ThirdPartyBots/PublicApiV1.md [hunk 1]
```diff
@@ -41,22 +41,37 @@ To identify the Third-Party Bots API endpoint user, first get domain information
...
+<ul>
+  <li>Bearer Token via Username/Password</li>
+  <li>Bearer Token via API (OAuth)</li>
+</ul>
+
+To perform login request you will need a valid Bot user created via LivePerson User Management UI. User must be `Enabled`, have minimum `Agent` role and have a **Login Method** selected either to `Password` or `API Key` as shown in Figure 2.1.
 
 {: .important}
-**Please note** We currently only support login with username/password and not with API method (i.e. OAuth). Moreover, only one session is maintained per user in LivePerson so if you receive bearer token error from Public API you can always generate another one by doing the login request again.
+**Please note** LivePerson maintain one session per user thus if you receive invalid bearer token error from Public API you can always generate new bearer by performing the login request again. We recommend making a single Bot Agent user that is dedicated for your Public API call.
```
Comment (erannnaot) [inline pages/documents/ConversationalAI/ThirdPartyBots/PublicApiV1.md]: *Please note** LivePerson maintains one session per user, thus if you receive an invalid bearer token error from Public API, you can always generate a new bearer by performing the login request again. We recommend making a single Bot Agent user that is dedicated for your Public API call.

**Review by akirilyuk — APPROVED** [review]

File: pages/documents/ConversationalAI/ThirdPartyBots/PublicApiV1.md [hunk 2]
```diff
@@ -41,22 +41,37 @@ To identify the Third-Party Bots API endpoint user, first get domain information
...
 
-To get the bearer token you must perform a `login` request to Third-Party Bots API Domain. To perform login request you will need a valid user created via LivePerson User Management UI. User must be `Enabled`, have minimum `Agent` role and **Login Method** set to `Password`.
+To use our Public API you must perform a login request to Third-Party Bots API domain which will in response send you a Third-Party Bots bearer token. There are two ways in which you can perform the login.
+
+<ul>
+  <li>Bearer Token via Username/Password</li>
+  <li>Bearer Token via API (OAuth)</li>
+</ul>
+
+To perform login request you will need a valid Bot user created via LivePerson User Management UI. User must be `Enabled`, have minimum `Agent` role and have a **Login Method** selected either to `Password` or `API Key` as shown in Figure 2.1.
```
Comment (erannnaot) [inline pages/documents/ConversationalAI/ThirdPartyBots/PublicApiV1.md]: To perform login requests you will need a valid Bot user created via LivePerson User Management UI. User must be `Enabled`, have minimum `Agent` role and have a **Login Method** selected either to `Password` or `API Key` as shown in Figure 2.1.

**Review by erannnaot — APPROVED** [review]
minor readability changes

**Pull request merged**